src/services/AttachmentApi.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * @copyright Copyright (c) 2018 Michael Weimann <mail@michael-weimann.eu>
+ *
+ * @author Michael Weimann <mail@michael-weimann.eu>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import axios from '@nextcloud/axios'
+
+export class AttachmentApi {
+
+	url(url) {
+		url = `/apps/deck${url}`
+		return OC.generateUrl(url)
+	}
+
+	async getAll(cardId) {
+		const response = await axios({
+			method: 'GET',
+			url: this.url(`/cards/${cardId}/attachments`),
+		})
+		return response.data
+	}
+
+	async createAttachment(data) {
```

Comment:
```suggestion
	async createAttachment({ cardId, formData}) {
```

Makes it a bit easier to actually get what is required here. Make sure to adjust the usages below as well then.